gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Project Settings
-dokka_version_base=1.4.0-rc
+dokka_version_base=0.11.0
```

Comment:
I think this was not an intended change, right?

Reply:
yes, definitely not